src/patternfly/components/ActionList/action-list.scss

```diff
@@ -0,0 +1,28 @@
+.pf-c-action-list {
+  --pf-c-action-list--Gap: var(--pf-global--spacer--md);
+  --pf-c-action-list--m-group--Gap: var(--pf-global--spacer--2xl);
+  --pf-c-action-list--m-group--m-vertical--Gap: var(--pf-global--spacer--lg);
+  --pf-c-action-list--m-vertical--Gap: var(--pf-global--spacer--sm);
+
+  &.pf-m-group {
```

Comment:
This requires all children of the action list to be groups. What if you want to have `action-list -> item + item + group + item`, with item spacing between items, and group spacing around groups?